### PR TITLE
K8SPXC-394 Check cluster-wide

### PR DESCRIPTION
**Pull request by hors**
[K8SPXC-394](https://jira.percona.com/browse/K8SPXC-394)

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| big-data | passed |
| recreate | passed |
| storage | passed |
| init-deploy | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| limits | failed |
| monitoring | passed |
| upgrade | failed |
| restore-to-encrypted-cluster | failed |
| upgrade-consistency | failed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| security-context | passed |
| self-healing-advanced | passed |
| affinity | passed |
| one-pod | passed |
| operator-self-healing | failed |
| auto-tuning | passed |
| scheduled-backup | failed |
| proxysql-sidecar-res-limits | passed |
| users | failed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| restore-to-encrypted-cluster | failed |
| init-deploy | passed |
| limits | failed |
| storage | passed |
| self-healing | passed |
| self-healing-advanced | passed |
| operator-self-healing | failed |
| big-data | passed |
| scaling | passed |
| scaling-proxysql | passed |
| upgrade | failed |
| monitoring | passed |
| monitoring-2-0 | passed |
| upgrade-consistency | failed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | failed |
| demand-backup | passed |
| scheduled-backup | failed |
| security-context | passed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| storage | passed |
| scaling | passed |
| recreate | passed |
| big-data | failed |
| scaling-proxysql | passed |
| self-healing | passed |
| limits | passed |
| monitoring | passed |
| upgrade | failed |
| restore-to-encrypted-cluster | failed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| self-healing-advanced | failed |
| affinity | passed |
| one-pod | passed |
| upgrade-consistency | failed |
| auto-tuning | passed |
| operator-self-healing | failed |
| security-context | passed |
| scheduled-backup | failed |
| proxysql-sidecar-res-limits | passed |
| users | failed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| limits | passed |
| monitoring | passed |
| storage | passed |
| monitoring-2-0 | passed |
| self-healing | passed |
| affinity | passed |
| scaling | passed |
| self-healing-advanced | passed |
| one-pod | passed |
| scaling-proxysql | passed |
| auto-tuning | passed |
| upgrade | failed |
| proxysql-sidecar-res-limits | passed |
| recreate | passed |
| users | failed |
| restore-to-encrypted-cluster | failed |
| big-data | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| operator-self-healing | failed |
| upgrade-consistency | failed |
| security-context | passed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| big-data | passed |
| scaling | passed |
| recreate | passed |
| init-deploy | passed |
| scaling-proxysql | passed |
| self-healing | failed |
| limits | passed |
| restore-to-encrypted-cluster | failed |
| self-healing-advanced | failed |
| upgrade | failed |
| monitoring | passed |
| demand-backup | passed |
| monitoring-2-0 | passed |
| operator-self-healing | passed |
| scheduled-backup | failed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| demand-backup-encrypted-with-tls | failed |
| proxysql-sidecar-res-limits | passed |
| users | failed |
| upgrade-consistency | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| big-data | passed |
| scaling | passed |
| storage | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| upgrade | failed |
| recreate | passed |
| restore-to-encrypted-cluster | failed |
| init-deploy | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | failed |
| demand-backup | passed |
| scheduled-backup | failed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| demand-backup-encrypted-with-tls | failed |
| upgrade-consistency | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| self-healing | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| scaling | passed |
| init-deploy | passed |
| scaling-proxysql | passed |
| limits | passed |
| upgrade | failed |
| monitoring | passed |
| recreate | passed |
| monitoring-2-0 | passed |
| restore-to-encrypted-cluster | failed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | failed |
| big-data | passed |
| demand-backup | passed |
| scheduled-backup | failed |
| demand-backup-encrypted-with-tls | failed |
| upgrade-consistency | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| big-data | passed |
| scaling | passed |
| scaling-proxysql | passed |
| upgrade | failed |
| init-deploy | passed |
| recreate | passed |
| limits | passed |
| restore-to-encrypted-cluster | failed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | failed |
| storage | passed |
| self-healing | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| demand-backup | passed |
| scheduled-backup | failed |
| demand-backup-encrypted-with-tls | failed |
| upgrade-consistency | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | failed |
| big-data | failed |
| init-deploy | failed |
| scaling | failed |
| storage | failed |
| restore-to-encrypted-cluster | failed |
| limits | failed |
| scaling-proxysql | failed |
| self-healing | failed |
| monitoring | failed |
| upgrade | failed |
| demand-backup | failed |
| self-healing-advanced | failed |
| upgrade-consistency | failed |
| monitoring-2-0 | failed |
| operator-self-healing | failed |
| scheduled-backup | failed |
| security-context | failed |
| affinity | failed |
| demand-backup-encrypted-with-tls | failed |
| one-pod | failed |
| auto-tuning | failed |
| proxysql-sidecar-res-limits | failed |
| users | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | failed |
| init-deploy | failed |
| scaling | passed |
| storage | failed |
| big-data | passed |
| self-healing | failed |
| limits | failed |
| monitoring | failed |
| scaling-proxysql | passed |
| monitoring-2-0 | failed |
| restore-to-encrypted-cluster | failed |
| self-healing-advanced | passed |
| upgrade | failed |
| demand-backup | failed |
| affinity | failed |
| one-pod | failed |
| auto-tuning | passed |
| scheduled-backup | failed |
| operator-self-healing | passed |
| demand-backup-encrypted-with-tls | failed |
| proxysql-sidecar-res-limits | failed |
| users | failed |
| upgrade-consistency | failed |
| security-context | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| init-deploy | passed |
| big-data | passed |
| scaling | passed |
| recreate | passed |
| self-healing | failed |
| scaling-proxysql | passed |
| limits | passed |
| self-healing-advanced | passed |
| restore-to-encrypted-cluster | failed |
| upgrade | failed |
| monitoring | passed |
| demand-backup | failed |
| scheduled-backup | failed |
| operator-self-healing | passed |
| demand-backup-encrypted-with-tls | failed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | failed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | failed |
| upgrade-consistency | failed |
| security-context | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| init-deploy | passed |
| recreate | passed |
| storage | passed |
| big-data | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| limits | passed |
| restore-to-encrypted-cluster | failed |
| upgrade | passed |
| monitoring | passed |
| self-healing-advanced | passed |
| upgrade-consistency | passed |
| security-context | failed |
| demand-backup | failed |
| monitoring-2-0 | passed |
| scheduled-backup | failed |
| affinity | passed |
| operator-self-healing | failed |
| one-pod | failed |
| auto-tuning | failed |
| proxysql-sidecar-res-limits | passed |
| users | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| storage | passed |
| recreate | passed |
| big-data | passed |
| init-deploy | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| limits | passed |
| upgrade | failed |
| restore-to-encrypted-cluster | passed |
| self-healing-advanced | failed |
| monitoring | passed |
| demand-backup | failed |
| operator-self-healing | failed |
| monitoring-2-0 | failed |
| scheduled-backup | failed |
| upgrade-consistency | passed |
| affinity | passed |
| security-context | failed |
| demand-backup-encrypted-with-tls | passed |
| one-pod | passed |
| auto-tuning | failed |
| proxysql-sidecar-res-limits | passed |
| users | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| storage | passed |
| init-deploy | passed |
| big-data | passed |
| recreate | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| limits | passed |
| upgrade | passed |
| restore-to-encrypted-cluster | passed |
| self-healing-advanced | passed |
| monitoring | passed |
| upgrade-consistency | passed |
| operator-self-healing | passed |
| demand-backup | passed |
| monitoring-2-0 | passed |
| security-context | failed |
| affinity | passed |
| one-pod | passed |
| scheduled-backup | failed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| demand-backup-encrypted-with-tls | passed |
| users | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | passed |
| storage | passed |
| big-data | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| limits | passed |
| upgrade | passed |
| restore-to-encrypted-cluster | passed |
| self-healing-advanced | failed |
| monitoring | passed |
| upgrade-consistency | passed |
| security-context | failed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| affinity | passed |
| scheduled-backup | failed |
| operator-self-healing | passed |
| one-pod | passed |
| auto-tuning | passed |
| demand-backup-encrypted-with-tls | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| recreate | passed |
| scaling | passed |
| init-deploy | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| limits | passed |
| upgrade | passed |
| restore-to-encrypted-cluster | passed |
| upgrade-consistency | passed |
| self-healing-advanced | passed |
| monitoring | passed |
| security-context | failed |
| demand-backup | passed |
| operator-self-healing | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| scheduled-backup | failed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| demand-backup-encrypted-with-tls | passed |
| users | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| storage | passed |
| big-data | passed |
| recreate | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| limits | passed |
| monitoring | failed |
| upgrade | passed |
| restore-to-encrypted-cluster | failed |
| self-healing-advanced | passed |
| upgrade-consistency | passed |
| monitoring-2-0 | failed |
| demand-backup | failed |
| security-context | passed |
| affinity | passed |
| one-pod | passed |
| operator-self-healing | passed |
| auto-tuning | passed |
| scheduled-backup | failed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| self-healing | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| scaling | passed |
| scaling-proxysql | passed |
| big-data | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |
| init-deploy | passed |
| limits | passed |
| monitoring | passed |
| recreate | passed |
| restore-to-encrypted-cluster | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| scheduled-backup | failed |
| demand-backup-encrypted-with-tls | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| big-data | passed |
| recreate | passed |
| init-deploy | passed |
| scaling | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| limits | passed |
| monitoring | passed |
| restore-to-encrypted-cluster | failed |
| upgrade | passed |
| self-healing-advanced | passed |
| monitoring-2-0 | passed |
| upgrade-consistency | passed |
| demand-backup | failed |
| affinity | passed |
| one-pod | failed |
| security-context | failed |
| auto-tuning | passed |
| scheduled-backup | failed |
| operator-self-healing | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| big-data | passed |
| storage | passed |
| recreate | passed |
| self-healing | passed |
| limits | passed |
| scaling-proxysql | passed |
| monitoring | passed |
| restore-to-encrypted-cluster | failed |
| upgrade | passed |
| upgrade-consistency | passed |
| monitoring-2-0 | passed |
| demand-backup | failed |
| security-context | failed |
| self-healing-advanced | passed |
| affinity | passed |
| one-pod | failed |
| operator-self-healing | passed |
| scheduled-backup | failed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| demand-backup-encrypted-with-tls | failed |
| users | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| init-deploy | passed |
| big-data | passed |
| storage | passed |
| recreate | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| limits | passed |
| restore-to-encrypted-cluster | failed |
| monitoring | passed |
| upgrade | passed |
| monitoring-2-0 | passed |
| self-healing-advanced | passed |
| upgrade-consistency | passed |
| demand-backup | failed |
| affinity | passed |
| security-context | failed |
| operator-self-healing | passed |
| one-pod | failed |
| scheduled-backup | failed |
| auto-tuning | passed |
| demand-backup-encrypted-with-tls | failed |
| proxysql-sidecar-res-limits | passed |
| users | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| big-data | passed |
| init-deploy | passed |
| recreate | passed |
| storage | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| limits | passed |
| upgrade | passed |
| restore-to-encrypted-cluster | failed |
| self-healing-advanced | passed |
| upgrade-consistency | passed |
| monitoring | passed |
| security-context | failed |
| demand-backup | failed |
| operator-self-healing | passed |
| monitoring-2-0 | passed |
| scheduled-backup | failed |
| affinity | passed |
| demand-backup-encrypted-with-tls | failed |
| one-pod | failed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| init-deploy | passed |
| scaling | passed |
| big-data | passed |
| recreate | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| limits | passed |
| monitoring | passed |
| upgrade | passed |
| restore-to-encrypted-cluster | failed |
| self-healing-advanced | passed |
| upgrade-consistency | passed |
| security-context | failed |
| monitoring-2-0 | passed |
| demand-backup | failed |
| affinity | passed |
| operator-self-healing | passed |
| one-pod | failed |
| scheduled-backup | failed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| demand-backup-encrypted-with-tls | failed |
| users | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| init-deploy | passed |
| storage | passed |
| big-data | passed |
| recreate | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| limits | passed |
| monitoring | passed |
| restore-to-encrypted-cluster | passed |
| upgrade | passed |
| monitoring-2-0 | passed |
| self-healing-advanced | passed |
| upgrade-consistency | passed |
| demand-backup | passed |
| security-context | passed |
| affinity | passed |
| one-pod | passed |
| operator-self-healing | passed |
| auto-tuning | passed |
| scheduled-backup | failed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| big-data | passed |
| scaling | passed |
| storage | passed |
| recreate | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| limits | passed |
| restore-to-encrypted-cluster | passed |
| monitoring | passed |
| upgrade | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| self-healing-advanced | passed |
| one-pod | passed |
| upgrade-consistency | passed |
| security-context | passed |
| auto-tuning | passed |
| demand-backup | passed |
| operator-self-healing | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| scheduled-backup | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| big-data | passed |
| recreate | passed |
| init-deploy | passed |
| storage | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| limits | passed |
| restore-to-encrypted-cluster | passed |
| monitoring | passed |
| upgrade | passed |
| monitoring-2-0 | passed |
| self-healing-advanced | passed |
| demand-backup | passed |
| upgrade-consistency | passed |
| security-context | passed |
| operator-self-healing | passed |
| affinity | passed |
| one-pod | passed |
| scheduled-backup | failed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| storage | passed |
| recreate | passed |
| big-data | passed |
| init-deploy | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| limits | passed |
| monitoring | passed |
| upgrade | passed |
| restore-to-encrypted-cluster | passed |
| upgrade-consistency | passed |
| monitoring-2-0 | passed |
| security-context | passed |
| demand-backup | passed |
| affinity | passed |
| self-healing-advanced | passed |
| one-pod | passed |
| scheduled-backup | failed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| operator-self-healing | passed |
| users | passed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| self-healing | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| big-data | passed |
| recreate | passed |
| restore-to-encrypted-cluster | passed |
| init-deploy | passed |
| demand-backup | passed |
| limits | passed |
| scheduled-backup | failed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| scaling | passed |
| auto-tuning | passed |
| scaling-proxysql | passed |
| proxysql-sidecar-res-limits | passed |
| upgrade | passed |
| users | passed |
| upgrade-consistency | passed |
| security-context | passed |
| demand-backup-encrypted-with-tls | failed |

File: deploy/cw_bundle.yaml [hunk 1]
```diff
@@ -0,0 +1,338 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```
Comment (delgod) [inline deploy/cw_bundle.yaml:1]: can we rename it to `deploy/cw-bundle.yaml`

File: deploy/cwrbac.yaml [hunk 2]
```diff
@@ -0,0 +1,116 @@
+kind: ClusterRole
```
Comment (delgod) [inline deploy/cwrbac.yaml:1]: can we rename it to `deploy/cw-rbac.yaml`

File: e2e-tests/functions [hunk 3]
```diff
@@ -68,9 +68,13 @@ create_namespace() {
...
 
 get_operator_pod() {
+    local ns=''
+    if [ -n "$OPERATOR_NS" ]; then
+        ns="-n $OPERATOR_NS"
+    fi
     kubectl_bin get pods \
         --selector=name=percona-xtradb-cluster-operator \
-        -o 'jsonpath={.items[].metadata.name}'
+        -o 'jsonpath={.items[].metadata.name}' $ns
```
Comment (delgod) [inline e2e-tests/functions:77]:
```suggestion
        -o 'jsonpath={.items[].metadata.name}' ${OPERATOR_NS:+-n $OPERATOR_NS}
```

File: e2e-tests/scheduled-backup/compare/cronjob_each-min-aws-s3.yml [hunk 4]
```diff
@@ -50,9 +50,9 @@ spec:
             - "\n\t\t\t\t\t\t\tcat <<-EOF | kubectl apply -f -\n\t\t\t\t\t\t\t\t\tapiVersion:
               pxc.percona.com/v1\n\t\t\t\t\t\t\t\t\tkind: PerconaXtraDBClusterBackup\n\t\t\t\t\t\t\t\t\tmetadata:\n\t\t\t\t\t\t\t\t\t
               \ name: \"cron-${pxcCluster:0:16}-$(date -u \"+%Y%m%d%H%M%S\")-${suffix}\"\n\t\t\t\t\t\t\t\t\t
-              \ labels:\n\t\t\t\t\t\t\t\t\t    ancestor: \"each-min-aws-s3\"\n\t\t\t\t\t\t\t\t\t
-              \   cluster: \"${pxcCluster}\"\n\t\t\t\t\t\t\t\t\t    type: \"cron\"\n\t\t\t\t\t\t\t\t\tspec:\n\t\t\t\t\t\t\t\t\t
-              \ pxcCluster: \"${pxcCluster}\"\n\t\t\t\t\t\t\t\t\t  storageName: \"aws-s3\"\n\t\t\t\t\t\t\tEOF\n\t\t\t\t\t\t\t"
+              \   ancestor: \"each-min-aws-s3\"\n\t\t\t\t\t\t\t\t\t    cluster: \"${pxcCluster}\"\n\t\t\t\t\t\t\t\t\t
```
Comment (delgod) [inline e2e-tests/scheduled-backup/compare/cronjob_each-min-aws-s3.yml:53]: looks like there are a mistake, `ancestor` should be the label, not metadata

File: e2e-tests/security-context/compare/statefulset_sec-context-proxysql-changes-oc.yml [hunk 5]
```diff
@@ -140,8 +140,8 @@ spec:
         runAsUser: 1001
         supplementalGroups:
         - 1001
-      serviceAccount: percona-xtradb-cluster-operator-workload
-      serviceAccountName: percona-xtradb-cluster-operator-workload
+      serviceAccount: default
```
Comment (delgod) [inline e2e-tests/security-context/compare/statefulset_sec-context-proxysql-changes-oc.yml:143]: we should use workload serviceAccount inside security-context test

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| big-data | passed |
| init-deploy | passed |
| recreate | passed |
| scaling | passed |
| storage | passed |
| self-healing | passed |
| limits | passed |
| restore-to-encrypted-cluster | failed |
| scaling-proxysql | passed |
| monitoring | passed |
| upgrade | passed |
| demand-backup | failed |
| monitoring-2-0 | passed |
| upgrade-consistency | passed |
| self-healing-advanced | passed |
| security-context | failed |
| scheduled-backup | failed |
| affinity | passed |
| one-pod | failed |
| demand-backup-encrypted-with-tls | failed |
| operator-self-healing | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| big-data | passed |
| scaling | passed |
| storage | passed |
| recreate | passed |
| init-deploy | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| limits | passed |
| restore-to-encrypted-cluster | passed |
| upgrade | passed |
| monitoring | passed |
| upgrade-consistency | passed |
| security-context | failed |
| self-healing-advanced | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | failed |
| demand-backup | passed |
| auto-tuning | passed |
| operator-self-healing | passed |
| scheduled-backup | failed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| demand-backup-encrypted-with-tls | passed |

File: e2e-tests/scheduled-backup/compare/cronjob_each-min-aws-s3.yml (hunk 4 above)
Comment (hors) [reply e2e-tests/scheduled-backup/compare/cronjob_each-min-aws-s3.yml:53]: fixed

File: e2e-tests/scheduled-backup/compare/cronjob_each-min-aws-s3.yml [hunk 6]
```diff
@@ -49,10 +49,10 @@ spec:
             - -c
             - "\n\t\t\t\t\t\t\tcat <<-EOF | kubectl apply -f -\n\t\t\t\t\t\t\t\t\tapiVersion:
               pxc.percona.com/v1\n\t\t\t\t\t\t\t\t\tkind: PerconaXtraDBClusterBackup\n\t\t\t\t\t\t\t\t\tmetadata:\n\t\t\t\t\t\t\t\t\t
-              \ name: \"cron-${pxcCluster:0:16}-$(date -u \"+%Y%m%d%H%M%S\")-${suffix}\"\n\t\t\t\t\t\t\t\t\t
-              \ labels:\n\t\t\t\t\t\t\t\t\t    ancestor: \"each-min-aws-s3\"\n\t\t\t\t\t\t\t\t\t
-              \   cluster: \"${pxcCluster}\"\n\t\t\t\t\t\t\t\t\t    type: \"cron\"\n\t\t\t\t\t\t\t\t\tspec:\n\t\t\t\t\t\t\t\t\t
-              \ pxcCluster: \"${pxcCluster}\"\n\t\t\t\t\t\t\t\t\t  storageName: \"aws-s3\"\n\t\t\t\t\t\t\tEOF\n\t\t\t\t\t\t\t"
+              -u \"+%Y%m%d%H%M%S\")-${suffix}\"\n\t\t\t\t\t\t\t\t\t  labels:\n\t\t\t\t\t\t\t\t\t
```
Comment (delgod) [inline e2e-tests/scheduled-backup/compare/cronjob_each-min-aws-s3.yml:52]: looks very strange, `name:` field disappeared

File: e2e-tests/security-context/conf/sec-context-changes.yml [hunk 7]
```diff
@@ -19,6 +19,7 @@ spec:
       supplementalGroups: [1001]
     size: 3
     image: -pxc
+    serviceAccountName: percona-xtradb-cluster-operator-workload
```
Comment (delgod) [inline e2e-tests/security-context/conf/sec-context-changes.yml:22]: sorry, can we also add it for ProxySQL?

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| big-data | passed |
| storage | passed |
| recreate | passed |
| init-deploy | passed |
| scaling | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| limits | passed |
| restore-to-encrypted-cluster | passed |
| upgrade | passed |
| self-healing-advanced | passed |
| upgrade-consistency | passed |
| security-context | passed |
| monitoring | passed |
| operator-self-healing | passed |
| demand-backup | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| scheduled-backup | failed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |

File: e2e-tests/security-context/conf/sec-context-changes.yml (hunk 7 above)
Comment (hors) [reply e2e-tests/security-context/conf/sec-context-changes.yml:22]: done

File: e2e-tests/scheduled-backup/compare/cronjob_each-min-aws-s3.yml (hunk 6 above)
Comment (hors) [reply e2e-tests/scheduled-backup/compare/cronjob_each-min-aws-s3.yml:53]: fixed

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| scaling | passed |
| recreate | passed |
| big-data | passed |
| init-deploy | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| limits | passed |
| restore-to-encrypted-cluster | passed |
| upgrade | passed |
| monitoring | passed |
| self-healing-advanced | passed |
| upgrade-consistency | passed |
| security-context | passed |
| monitoring-2-0 | passed |
| operator-self-healing | passed |
| demand-backup | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| scheduled-backup | failed |
| proxysql-sidecar-res-limits | passed |
| users | passed |

**Pull request merged**